codeclimate-wrapper

```diff
@@ -41,17 +41,31 @@ else
     && invalid_docker_host "$DOCKER_HOST"
 fi
 
+: ${XDG_CONFIG_HOME:=$HOME/.config/}
+CC_CONFIG=$(realpath "${CODECLIMATE_CONFIG:-$XDG_CONFIG_HOME/codeclimate/config.yml}")
```

Comment:
What's the portability of `realpath`? We claim the wrapper should work on any POSIX system.